### DISCUSSION TITLE
ideas and code sharing

### DISCUSSION THREAD

**Discussion by manigamer22** [lysep-corp/ButOSX#21] [Ideas]
Show i got some codes working to my self but why not share it with the public? So I would like to request the codes you have developed for this cheat to be added and shared to the public aswell.

**Comment by Lyceion** [comment]
> Show i got some codes working to my self but why not share it with the public? So I would like to request the codes you have developed for this cheat to be added and shared to the public aswell. 

Yes you can. Just download the latest code from BETA and make a pull request for BETA. You know, i'm adding codes to beta first for security purposes...
